### PR TITLE
add files for review

### PR DESCRIPTION
**Pull request by fpsapc**
add files for review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
     <footer id="footer"><a>Click Here</a></footer>
 </body>
 </html>
-<!-- When you click a .nav-link button in the nav element, you are taken to the corresponding section of the landing page
+<!-- When ou click a .nav-link button in the nav element, you are taken to the corresponding section of the landing page
 You can watch an embedded product video with id="video"
 Your landing page has a form element with a corresponding id="form"
 Within the form, there is an input field with id="email" where you can enter an email address
```
Comment (fpsapc) [inline index.html]: Excellent work done so far. Indeed you have spent time on this project! 👏
Right use of Git Flow ✨
There are some issues that you still need to work on to go to the next project but you are almost there!
Highlights:
Good Commit Messages.
Well Organized Readme File.
Required Changes:
I suggest to remove the comments from your code.

**Review by fpsapc — COMMENTED** [review]
Hi @fpsapc ,
Congratulations! 🎉 Your project is complete and ready to be merged! 

**Highlights:**
Great job on getting everything done, you should be proud of your hard work! 👏

**Review by nahnahdev — CHANGES_REQUESTED** [review]
Hi @fpsapc , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [x] Good use of inline comments

## Required Changes ♻️
- [x]  I will recommend highlighting what the student did good with the project so far, you can mention things like the good practices the student applied to the project, check if he followed GitHub flow correctly..etc

- [x] I also suggest you add emojis or GIFs to your comments  to help give your comments a more friendly tone :wink: 

- [x]  I also strongly suggest that in order to sound polite you need to be using words like `please, kindly, I suggest, I highly recommend .`..etc, and many other words you may find polite :100: 

- [x]  Please explain why you are asking for that change by providing examples and best practices approach so the student can learn something :man_teachersmile: as well
- [x] Using the feedback above, Kindly add an additional comment as in the project requirements, you were asked to write at least 2 comments :heavy_check_mark: 
- [x] Kindly make some changes to your comment above as well :100: 
- [x] Your new comment can be about Html and CSS best practices, or lack of [semantic HTML tags use](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), etc. and ensure that you add why you are requesting the change :100:  You can introduce bugs in your codebase if you can't find anything to talk about and then point it out in your review :+1:


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -30,15 +30,4 @@
     </main>
     <footer id="footer"><a>Click Here</a></footer>
 </body>
-</html>
```
Comment (fpsapc) [inline index.html:33]: @NahnahAJ  Please use html tag to close the already open `<html>` tag. Kindly put `</html>` tag again in your code.

File: index.html [hunk 3]
```diff
@@ -30,15 +30,4 @@
...
-You can watch an embedded product video with id="video"
-Your landing page has a form element with a corresponding id="form"
-Within the form, there is an input field with id="email" where you can enter an email address
-The #email input field should have placeholder text to let users know what the field is for
-The #email input field uses HTML5 validation to confirm that the entered text is an email address
-Within the form, there is a submit input with a corresponding id="submit"
-When you click the #submit element, the email is submitted to a static page (use this mock URL: https://www.freecodecamp.com/email-submit)
-The navbar should always be at the top of the viewport
-Your product landing page should have at least one media query
-Your product landing page should utilize CSS flexbox at least once -->
```
Comment (fpsapc) [inline index.html:44]: Great Job. Work done very well.

**Review by fpsapc — COMMENTED** [review]
Excellent work done so far. Indeed you have spent time on this project! :clap:
Right use of  Git Flow :sparkles:
There are some issues that you still need to work on to go to the next project but you are almost there!
**Highlights:**
Good Commit Messages.
Well Organized Readme File. 
**Required Changes:**
Please Check the comments under the review. @NahnahAJ Sir please review my comments now. Thank You

**Comment by NduatiKagiri** [comment]
Hi @fpsapc 👋 ,

# APPROVED 💯  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ea0cb5a48e3337523ffe8d60f177879d

## Highlights ✔️ 
✅ Highlights well put
✅ Polite words to request changes 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**